tests/data-files/m_3211605_ne_11_060_20200415_20200730.xml

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+	<idinfo>
+		<citation>
+			<citeinfo>
+				<origin>USDA-Farm Production and Conservation Business Center (FPAC-BC)</origin>
+				<pubdate>20200729</pubdate>
+				<title></title>

```

Comment:
these two xml files are the same. Is this one supposed to have the title value set?

Reply:
Added to have create item pull resource_desc and dt from cog_href (not xml). This wasn't included in tests and did throw an error whenever tested.